snc.sh

```diff
@@ -332,8 +332,8 @@ fi
 ${OPENSHIFT_INSTALL} --dir ${INSTALL_DIR} destroy cluster ${OPENSHIFT_INSTALL_EXTRA_ARGS} || echo "failed to destroy previous cluster.  Continuing anyway"
 # Generate a new ssh keypair for this cluster
 
-rm id_rsa_crc* || true
-ssh-keygen -N "" -f id_rsa_crc -C "core"
+rm id_ecdsa_crc* || true
+ssh-keygen -t ecdsa -b 521 -N "" -f id_ecdsa_crc -C "core"
```

Comment:
out of curiosity, why -b 521? why ecdsa rather than ed25519? (could be in the commit log)

Reply:
No real reason for ecdsa over ed25519...  ECDSA has been around a little longer, but still considered solid.

Since 521 bits is the largest key size, but still not big enough to rally impact anything, I used 521 over 256 or 384.  Again, no real reason.  I'll add a comment in the script.